src/vehicles/RosVehicle.lua

```diff
@@ -56,9 +56,18 @@ function RosVehicle:onLoad()
     local spec = self.spec_rosVehicle
     -- rosVehicle variables
     -- .id is initialized by FS
-    spec.ros_veh_name = ros.Names.sanatize(self:getFullName() .. "_" .. self.id)
-    spec.base_link_frame = spec.ros_veh_name .. "/base_link"
-
+    spec.ros_veh_name_w_id = ros.Names.sanatize(self:getFullName() .. "_" .. self.id)
+    spec.ros_veh_name = ros.Names.sanatize(self:getFullName())
```

Comment:
Maybe add a comment explaining why we have 'two names'?

Reply:
Added!